SharedPackages/BrowserServicesKit/Sources/Networking/Auth/OAuthClient.swift

```diff
@@ -27,6 +27,7 @@ public enum OAuthClientError: Error, LocalizedError, Equatable {
     /// When both access token and refresh token are expired
     case refreshTokenExpired
     case invalidTokenRequest
+    case authMigrationNotPerformed
```

Comment:
I understand the reason for adding this but when reading it this does not sound like a real error, just a state or result that migration was skipped as it was not necessary.